.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules
 client/.pnp
 client/.pnp.js
 client/build
+!client/.env
```

Comment:
Only one of these changes is required to keep `client/.env` tracked.

Reply:
Not sure if I understood your comment, I tried `.env` for the root one, and didn’t work. I suppose if you don't specify `/` before the filename it will ignore all occurrences in the project.